src/index.html

```diff
@@ -17,6 +17,79 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="Moyo logo"
+      />
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              href="index.html"
+              class="link link-1"
+            >
+              VIDEO
```

Comment:
[checklist](https://github.com/mate-academy/layout_moyo-header?tab=readme-ov-file#checklist): CSS is used to show all letters in Uppercase (don't type them in HTML)